gen/src/download/mod.rs

```diff
@@ -8,107 +8,86 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+
+//! UNIC - Data Download Script
+//!
+//! # Config Format:
+//!
+//! ```toml
+//! [example]
+//! version = "10.0.0"
+//! base_url = "http://example.com/path/to/{version}/"
+//!
+//! [[example.resources]]
+//! "some_data.txt" = "some_data.txt"
+//!
+//! [[example.test_resources]]
```

Comment:
You changed the `.` to a `_` but missed the fact that you've double bracketed the headers here incorrectly.

Maybe that's not as obvious a syntax as TOM thinks it is 😛

Reply:
(disclaimer: that was a joke by a tired brain)

Reply:
🤣 Yeah, that one is **really** hard to see!!!